cartography/data/jobs/analysis/gcp_sql_instance_analysis.json

```diff
@@ -5,9 +5,27 @@
             "iterative": true,
             "iterationsize": 1000
         },
+        {
+            "query": "MATCH (u:GCPSQLUser{exposed_internet:true})<-[r:USED_BY]-(:GCPSQLInstance})<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WITH u LIMIT $LIMIT_SIZE REMOVE u.exposed_internet, u.exposed_internet_type return COUNT(*) as TotalCompleted",
```

Comment:
public facing logic does not apply to GCPSQLUser object. Please remove this code from this file.